query/src/servers/http/v1/query/execute_state.rs

```diff
@@ -196,6 +199,9 @@ impl ExecuteState {
                 }
                 log::debug!("drop block sender!");
             })?;
+
+        // Write Finish to query log table.
+        interpreter.finish().await?;
```

Comment:
@BohuTANG  query is not finished here 
https://github.com/BohuTANG/datafuse/pull/21

Reply:
Thank you, some unit tests failure related to it , I am trying to fix